Yoast/wordpress-seo
# WordProof UI fixes

## Context

* Fixing bugs found in RC

## Summary

This PR can be summarized in the following changelog entry:

- Fixes [QA-3342:](https://yoast.atlassian.net/browse/QA-3342)
- Fixes [QA-3345:](https://yoast.atlassian.net/browse/QA-3345)

## Relevant technical choices:

* I made use of the admin asset manager for enqueuing a script in the front end. This is done semi-manually in the `wordproof.php` file as the register_scripts is executed in the `admin_init` action.

## Test instructions

#3 [QA-3345](https://yoast.atlassian.net/browse/QA-3345)
- Set up clean multisite live env (subdirectory)
- Install and NETWORK activate Yoast Free 19.2 RC-3
- Install and NETWORK activate WordProof plugin
- Go to subsite> Yoast SEO> General> Integrations
- Check msg near WordProof integration

Expected result: “The WordProof Timestamp plugin needs to be disabled before you can activate this integration.”

#1 QA-3342:
- Set up clean live env
- Install and activate Yoast Free 19.2 RC-3
- Turn ON WordProof integration
- Go to the Privacy page and connect WordProof account
- Timestamp the Privacy page
- Check frontend of the Privacy page as Admin (Actual result: WordProof badge is shown)
- Open the Privacy Page in an incognito tab as a site visitor

Expected result: A WordPoof badge.

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

See test instructions.

### Test instructions for QA when the code is in the RC



## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

* the WordProof integration

## UI changes


## Other environments


## Documentation


## Quality assurance


Fixes #
